pkg/bytecode/vm_test.go

```diff
@@ -118,6 +118,22 @@ func TestBoolExpressions(t *testing.T) {
 				mustMake(t, OpSetGlobal, 0),
 			},
 		},
+		{
+			name: "not operator",
+			input: `
+			x := !true
+			x = x
```

Comment:
Consider a clean-up follow-up PR that removes all these `x = x` from the test cases as well as the `GetGlobal` / `SetGlobal` at the end of the `expectedInstructions` and instead have the test runner add these. This is how the evaluator works: https://github.com/evylang/evy/blob/e9e8840bba3957d00fd405af7b94929fe12eddd9/pkg/evaluator/evaluator_test.go#L53 in many places. Then the test input is a simple one-liner string in many cases and is generally cleaner.